### ISSUE TITLE
facebook-ios-sdk v8.2.0 memory leaks

### ISSUE THREAD

**Issue by hw20101101** [facebook/facebook-ios-sdk#1608]
## Checklist

- [x] I've updated to the latest released version of the SDK
- [x] I've searched for existing [GitHub issues](https://github.com/facebook/facebook-ios-sdk/issues)
- [x] I've looked for existing answers on [Stack Overflow](https://facebook.stackoverflow.com), the [Facebook Developer Community Forum](https://developers.facebook.com/community/) and the [Facebook Developers Group](https://www.facebook.com/groups/fbdevelopers)
- [x] I've read the [Code of Conduct](https://github.com/facebook/facebook-ios-sdk/blob/master/CODE_OF_CONDUCT.md)
- [x] This issue is not security related and can safely be disclosed publicly on GitHub

## Environment

Describe your dev environment here, giving as many details as possible. If you have them, make sure to include:

- Xcode Version: `12.2`
- Swift Version: `5.3` (if issue is Swift related)
- Installation Platform & Verison: `Cocoapods 1.10.1`

## Goals

What do you want to achieve?

facebook login

## Expected Results

What do you expect to happen?

No memory leaks

## Actual Results

What actually happened? Can you provide a stack trace?

Memory leak，I have a screenshot of the detection of Instruments

## Steps to Reproduce

What are the steps necessary to reproduce this issue?

Initialize facebook sdk in Appdelegate of oc project

## Code Samples & Details

Please provide a code sample, as well as any additional details, to help us track down the issue. If you can provide a link to a test project that allows us to reproduce the issue, this helps us immensely in both the speed and quality of the fix.

**Note:** Remember to format your code for readability:

```  
// INSERT YOUR CODE HERE

- (BOOL)application:(UIApplication*)application didFinishLaunchingWithOptions:(NSDictionary*)launchOptions
{

    [[FBSDKApplicationDelegate sharedInstance] application:application
                             didFinishLaunchingWithOptions:launchOptions];
    
    return YES;

}

```
![WX20210111-152626@2x](https://user-images.githubusercontent.com/9107226/104160677-7a2d1080-542c-11eb-803c-25e2c66b8f36.png)

--- 
--- 

![WX20210111-152647@2x](https://user-images.githubusercontent.com/9107226/104160685-7dc09780-542c-11eb-9188-db71cd0c65ba.png)

**Comment by dourgulf** [comment]
Very much leaks reported by Xcode instrument！

**Comment by hhl1809** [comment]
How to solve this issue ? I got much leaks from FBSDK too.

**Comment by joesus** [comment]
Thanks for bringing this to our attention!

Glad to announce the fixes for these have been merged and we plan to release them in 9.1.0. 

757bbec07d14e0437b7e1b177fda90c5491d80b8
b35383a811154c5ccd6e92946f26953954c8b814
067319838bd25a0f139862e7596fa853301dea54

Thanks!
